action.yml

```diff
@@ -17,6 +17,9 @@ inputs:
   force-push:
     description: 'Force push to Platform.sh'
     required: false
+  instance-name:
+    description: 'Name of the platform.sh instance'
+    default: ${GITHUB_REF_NAME}
```

Comment:
Shouldn't this be `${{ ... }}`? Also, do we know that this substitution works in this section of the yml file?

Reply:
Yes. I am going to try this branch in one of my projects.

Reply:
Well, none of them are working. They are just printing the exact text `${GITHUB_REF_NAME}` if you don't pass the variable.
The documentation doesn't explicitly mentions that you can use expression, but you can use plain string.
I am trying to set the default using output parameters.

Reply:
It is now working with step output. However, it is not considering the environment name passed in the ci yaml file. I will take a look.

Reply:
The way I was passing the environment name was incorrect. That is why it was not getting considered.